extensions/windows-event-log/wel/MetadataWalker.cpp

```diff
@@ -171,4 +176,17 @@ void MetadataWalker::updateText(pugi::xml_node &node, const std::string &field_n
   }
 }
 
+void MetadataWalker::updateText(pugi::xml_attribute &node, const std::string &field_name, std::function<std::string(const std::string &)> &&fn) {
```

Comment:
I think we should pass `fn` without type erasure, since it's all local to this class.